### PR TITLE
Add `Table` component

### PR DESCRIPTION
**Pull request by toha993**

### CODE REVIEW

File: packages/react/src/index.ts [hunk 1]
```diff
@@ -56,6 +56,12 @@ export * from "./skeleton";
 export * from "./sprinkles";
 export * from "./styles";
 export * from "./switch";
+export * from "./table";
```
Comment (jaf107) [inline packages/react/src/index.ts:59]: Let's move the components to unstable.ts file & remove the web-components for now

File: packages/react/src/table/Table.css.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export const table = recipe({
+  base: [
+    {
+      w: "full",
+    },
+    style({
+      borderRadius: "5px",
+      borderSpacing: "0px",
+      captionSide: "bottom",
+      fontSize: "14px",
```
Comment (jaf107) [inline packages/react/src/table/Table.css.ts:12]: We can use this in sprinkles

File: packages/react/src/table-cell/TableCell.css.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      p: "12",
+    },
+    style({
+      ":last-of-type": {
+        borderRight: "0",
+      },
+      backgroundColor: "white",
+      borderBottom: `1px solid ${theme.colors["gray.200"]}`,
+      borderRight: `1px solid ${theme.colors["gray.200"]}`,
+      color: theme.colors["gray.600"],
```
Comment (jaf107) [inline packages/react/src/table-cell/TableCell.css.ts:20]: Let's use sprinkles for this

File: packages/react/src/table-row/TableRow.css.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { recipe, style } from "../vanilla-extract";
+
+export const row = recipe({
+  base: [
+    style({
+      width: "100%",
```
Comment (jaf107) [inline packages/react/src/table-row/TableRow.css.ts:6]: Let's use sprinkles for this

Comment (jaf107) [reply packages/react/src/table-row/TableRow.css.ts:6]:
```suggestion
      w: "full",
```

File: packages/react/src/table-cell/TableCell.css.ts (hunk 3 above)
Comment (jaf107) [reply packages/react/src/table-cell/TableCell.css.ts:20]:
```suggestion
      color: "gray.600",
```

**Review by radnan — APPROVED** [review]

**Pull request merged**